docs/components/store.md

```diff
@@ -212,6 +212,7 @@ config:
   max_idle_connections: 0
   max_async_concurrency: 0
   max_async_buffer_size: 0
+  max_item_size: 0
```

Comment:
Would it make sense for this option to be available also for in-memory cache?

Reply:
Yes, it's already available:
https://github.com/thanos-io/thanos/blob/d89a497d03a3d41781c26cf6f248f214f08bace7/pkg/store/cache/inmemory.go#L57